internal/exec/engine.go

```diff
@@ -45,6 +48,9 @@ import (
 
 const (
 	DefaultFetchTimeout = 2 * time.Minute
+	// This variable will help to identify ignition journal messages
+	// related to the user/base config.
+	MessageID = "57124006b5c94805b77ce473e92a8aeb"
```

Comment:
Hmm, we don't need to export this constant, right? Maybe also something more descriptive like `ignitionFetchedConfigMsgId` ?